src/common/definitions.js

```diff
@@ -311,6 +311,21 @@ const educationProgram = {
   }
 };
 
+const currentlyActiveDuty = {
+  type: 'object',
+  properties: {
+    yes: {
+      type: 'boolean'
+    },
+    onTerminalLeave: {
+      type: 'boolean'
+    },
+    nonVaAssistance: {
```

Comment:
Ok. I'll check with Alex and see if the UI needs to change, since I don't think we're limiting that field to active duty vets only.